src/Button/RedliningButton/RedliningButton.jsx

```diff
@@ -0,0 +1,672 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Modal, Input } from 'antd';
+
+import OlMap from 'ol/map';
+import OlLayerVector from 'ol/layer/vector';
+import OlSourceVector from 'ol/source/vector';
+import OlCollection from 'ol/collection';
+import OlStyleStyle from 'ol/style/style';
+import OlStyleStroke from 'ol/style/stroke';
+import OlStyleFill from 'ol/style/fill';
+import OlStyleCircle from 'ol/style/circle';
+import OlStyleText from 'ol/style/text';
+import OlInteractionDraw from 'ol/interaction/draw';
+import OlInteractionSelect from 'ol/interaction/select';
+import OlInteractionModify from 'ol/interaction/modify';
+import OlInteractionTranslate from 'ol/interaction/translate';
+import OlEventsCondition from 'ol/events/condition';
+
+import ToggleButton from '../ToggleButton/ToggleButton.jsx';
+import MapUtil from '../../Util/MapUtil/MapUtil';
+import StringUtil from '../../Util/StringUtil/StringUtil';
+import DigitizeUtil from '../../Util/DigitizeUtil/DigitizeUtil';
+import Logger from '../../Util/Logger';
+
+import './RedliningButton.less';
+
+/**
+ * The RedliningButton.
+ *
+ * @class The RedliningButton
+ * @extends React.Component
+ *
+ */
+class RedliningButton extends React.Component {
```

Comment:
Not technical, but could we please get rid of the term `Redlining` (see [here](https://en.wikipedia.org/wiki/Redlining)).

Reply:
Since 938d625 the class is renamed to `DigitizeButton`